WooCommerce/Classes/Extensions/String+Helpers.swift

```diff
@@ -101,6 +101,14 @@ extension String {
 
         return Set(arrayOfTags)
     }
+
+    func removingSuffix(_ suffix: String) -> String {
+        guard hasSuffix(suffix) else {
+            return self
+        }
+
+        return String(self.dropLast(suffix.count))
+    }
```

Comment:
This implementation is similar to how standard Kotlin version works, so if there is a better way to handle this please let me know.

Reply:
Looks like there's an existing method for removing suffix in the Networking framework [here](https://github.com/woocommerce/woocommerce-ios/blob/89962fe408655647e01ff8e8ead84480c4e2a882/Networking/Networking/Extensions/String%2BHelpers.swift#L84). Since the `AIProductMapper` file targets Networking, calling `removingSuffix` would reference that method instead of this one. I think this method is not needed (for now).